Apps/phironnet/ironnet_connector.py

```diff
@@ -353,6 +409,26 @@ def initialize(self):
         self._password = config.get('password')
         self._verify_server_cert = config.get('verify_server_cert')
 
+        # Alert Notification Configs
+        self._enable_alert_notifications = config.get('enable_alert_notifications')
+        alert_cats = config.get('alert_categories')
+        if alert_cats:
+            self._alert_categories = [str(cat).replace(" ", "_").upper() for cat in alert_cats.split(',')]
+        else:
+            self._alert_categories = []
+        alert_subcats = config.get('alert_subcategories')
+        if alert_subcats:
+            self._alert_subcategories = [str(subcat).replace(" ", "_").upper() for subcat in alert_subcats.split(',')]
```

Comment:
We should also trim whitespace off each parsed alert subcategory to minimize user error.